website/src/routes/api/(schemas)/string/index.mdx

```diff
@@ -2,6 +2,116 @@
 title: string
 ---
 
+import { Property } from '~/components';
+import { Link } from '@builder.io/qwik-city';
+
 # string
 
-> The content of this page is not yet ready. Until then just use the [source code](https://github.com/fabian-hiller/valibot/blob/main/library/src/schemas/string/string.ts).
+Schema that parses the input to string.
+
+```ts
+const schema = string(errorMessage, pipe);
+```
+
+## Parameter
+
+- `errorMessage` <Property type='string'/> = 'Invalid type'
+- `pipe` <Property type={{ type: 'custom', name: 'Pipe<string>' }}/> = []
```

Comment:
The syntax `name: 'Pipe<string>'` is not quite correct:

```ts
const type = {
  type: 'custom',
  name: 'Pipe',
  href: '../Pipe',
  generics: ['string'],
};
```

Reply:
To not mess up the mdx file, you can also define complex properties at the end of the file like I did in Modular Forms: https://github.com/fabian-hiller/modular-forms/blob/main/website/src/routes/(layout)/[framework]/api/getValues.mdx

Reply:
I would be interested to know how other documentation handles this internally. Maybe we can use that to improve our approach with `<Property />`.

Reply:
Implemented!
https://github.com/fabian-hiller/valibot/pull/220/commits/a224502ecd201223d41674a6d2138702efea9504

> I would be interested to know how other documentation handles this internally. Maybe we can use that to improve our approach with <Property />.

I'll look this up in the coming days 🙏

Reply:
> I would be interested to know how other documentation handles this internally.

I've searched 3 library documents, and found that there are only simple approaches to handle type and property information:
- React Hook Form
  - only code block is used, and there is no component like `<Property />`
  - https://react-hook-form.com/ts#Field
- Material UI
  - only code block and  `` block are used, and there is no component like `<Property />`
  - https://mui.com/material-ui/api/accordion/
- Typia
  - only code block and  `` block are used, and there is no component like `<Property />`
  - https://typia.io/docs/validators/validate/

So as far as I've searched, I guess it might be good to use <Property /> without big changes.

Reply:
Thank you for the research!